File: gex/Models/Db/BarMatchChatMessage.cs
using Dapper.ColumnMapper;
using gex.Code;

namespace gex.Models.Db {

    [DapperColumnsMapped]
    public class BarMatchChatMessage {

        [ColumnMapping("game_id")]
        public string GameID { get; set; } = "";

        [ColumnMapping("game_timestamp")]
        public float GameTimestamp { get; set; }

        [ColumnMapping("size")]
        public byte Size { get; set; }

        [ColumnMapping("from_id")]
        public byte FromId { get; set; }

        [ColumnMapping("to_id")]
        public byte ToId { get; set; }

        [ColumnMapping("message")]
        public string Message { get; set; } = "";

    }
}

File: gex/src/model/BarMatchChatMessage.ts

export class BarMatchChatMessage {
    public gameID: string = "";
    public gameTimestamp: number = 0;
    public fromId: number = 0;
    public toId: number = 0;
    public message: string = "";

    public static parse(elem: any): BarMatchChatMessage {
        return {
            ...elem
        };
    }

}